src/main/kotlin/org/digma/intellij/plugin/navigation/codeless/CodelessNavigation.kt

```diff
@@ -0,0 +1,89 @@
+package org.digma.intellij.plugin.navigation.codeless
+
+import com.intellij.openapi.components.service
+import com.intellij.openapi.diagnostic.Logger
+import com.intellij.openapi.project.Project
+import org.digma.intellij.plugin.document.CodeObjectsUtil
+import org.digma.intellij.plugin.editor.CurrentContextUpdater
+import org.digma.intellij.plugin.log.Log
+import org.digma.intellij.plugin.model.discovery.CodeLessSpan
+import org.digma.intellij.plugin.service.ErrorsActionsService
+import org.digma.intellij.plugin.ui.service.ErrorsViewService
+import org.digma.intellij.plugin.ui.service.InsightsViewService
+
+
+private val logger = Logger.getInstance("org.digma.intellij.plugin.navigation.codeless.CodelessNavigation")
+
+fun navigate(
```

Comment:
not must by I think we should have a class that represent SpanLocation which contains span name, instrumentation, namespace and function and public method to get the span_id and method_id if exists.
Also, it will deal with all the parsing  $_$.....